eng/pipelines/prepare-pipelines.yml

```diff
@@ -0,0 +1,10 @@
+trigger: none
+
+extends:
+  template: /eng/common/pipelines/templates/steps/prepare-pipelines.yml
+  parameters:
+    Repository: $(Build.Repository.Name)
+    Prefix: cpp
+    CIConventionOptions: ''
+    UPConventionOptions: '--variablegroup 58 --variablegroup 76 --variablegroup 56'
```

Comment:
What do these values mean? Also, what is UP?

Reply:
```up``` means _Unified Pipeline_. It ties back to a convention supported in Pipeline Generator (the tool we use to auto generate your pipelines from ```ci.yml``` files.

Reply:
The ```--variablegroup``` arguments relate the IDs of variable groups in Azure Pipelines that we map to the pipeline. For example the release pipelines need access to certain secrets which are exposed in variable groups.